### PR TITLE
Literature: make list view more compact

### PR DESCRIPTION
**Pull request by drjova**
* Makes Literature list view more compact and optimized for
  smaller screens.

Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: ui/src/literature/components/LiteratureDate/LiteratureDate.jsx [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+
 class LiteratureDate extends Component {
   render() {
-    const { date } = this.props;
+    const { date, wrapperClassName } = this.props;
 
     return (
-      <span>{date}</span>
+      <span className="__LiteratureDate__">
+        <span className={wrapperClassName}>{date}</span>
```
Comment (harunurhan) [inline ui/src/literature/components/LiteratureDate/LiteratureDate.jsx]: this is confusing, and unnecessary to push down to this component just for displaying date with paranthesis. I think you can just wrap it with paranthesis in `jsx markup` wherever you want.

File: ui/src/literature/components/__tests__/LiteratureItem.test.jsx [hunk 2]
```diff
@@ -14,7 +14,6 @@ describe('LiteratureItem', () => {
       control_number: 12345,
       citation_count: 1,
       publication_info: [{ journal_title: 'Test Jornal' }],
-      dois: [{ value: '12345679.1234.123' }],
       abstract: [{ value: 'Test Abstract' }],
```
Comment (harunurhan) [inline ui/src/literature/components/__tests__/LiteratureItem.test.jsx]: should be removed too.

File: ui/src/literature/components/LiteratureDate/LiteratureDate.jsx (hunk 1 above)
Comment (harunurhan) [inline ui/src/literature/components/LiteratureDate/LiteratureDate.jsx]: this is confusing to push into this component, just to display it with parenthesis in some places. You could just wrap the component in jsx markup with parenthesis wherever you want.

File: ui/src/common/layouts/SearchLayout/SearchLayout.jsx [hunk 3]
```diff
@@ -32,12 +32,12 @@ class SearchLayout extends Component {
...
     return (
       <Row className="__SearchLayout__" gutter={32} type="flex" justify="start">
-        <Col xs={0} lg={8} xl={6} xxl={5}>
+        <Col xs={0} lg={8} xl={7} xxl={5}>
           {!withoutAggregations && (
             <ResponsiveView min="lg" render={() => this.renderAggregations()} />
           )}
         </Col>
-        <Col xs={24} lg={16} xl={15} xxl={14}>
+        <Col xs={24} lg={16} xl={16} xxl={14}>
```
Comment (harunurhan) [inline ui/src/common/layouts/SearchLayout/SearchLayout.jsx]: How about embedded search ? It may need to be changed too.

File: ui/src/App.scss [hunk 4]
```diff
@@ -9,6 +9,10 @@
       margin-top: 0px !important;
       padding: $header-padding;
     }
+
+    @media (max-width: $screen-lg-max) {
+      padding: 0 1rem;
```
Comment (harunurhan) [inline ui/src/App.scss]: maybe you could use the `xs` one already which is `12px` just `2` less than this.

File: ui/src/styleVariables.js [hunk 5]
```diff
@@ -6,9 +6,10 @@ module.exports = {
   'link-hover-color': '#40a9ff',
   'layout-body-background': '#f0f2f5',
   'success-color': '#52c41a',
-  'secondary-color': 'rgba(0, 0, 0, 0.45)',
+  'secondary-color': 'rgba(0, 0, 0, 0.8)',
   'screen-xs-max': '575px',
   'screen-sm-max': '767px',
+  'screen-lg-max': '159px',
```
Comment (harunurhan) [inline ui/src/styleVariables.js]: this is wrong

File: ui/src/literature/components/LiteratureDate/LiteratureDate.jsx (hunk 1 above)
Comment (drjova) [reply ui/src/literature/components/LiteratureDate/LiteratureDate.jsx]: it looked uglier to me doing this though but  👍

File: ui/src/App.scss (hunk 4 above)
Comment (drjova) [reply ui/src/App.scss]: 👍 done

File: ui/src/common/layouts/SearchLayout/SearchLayout.jsx (hunk 3 above)
Comment (drjova) [reply ui/src/common/layouts/SearchLayout/SearchLayout.jsx]: it looks ok in the embed search

File: ui/src/authors/containers/DetailPage.jsx [hunk 6]
```diff
@@ -67,7 +67,7 @@ class DetailPage extends Component {
     return (
       <Fragment>
         <Row type="flex" justify="center">
-          <Col className="mv3" xs={24} md={21} lg={19} xl={18}>
+          <Col className="mv2" xs={24} md={21} lg={19} xl={18}>
```
Comment (harunurhan) [inline ui/src/authors/containers/DetailPage.jsx]: How about literature?

File: ui/src/literature/components/LiteratureItem.jsx [hunk 7]
```diff
@@ -61,25 +67,27 @@ class LiteratureItem extends Component {
...
-          <LiteratureDate date={date} />
+        <div className="mt1">
+          <AuthorsAndCollaborations
+            authorCount={authorCount}
+            authors={authors}
+            collaborations={collaborations}
+            collaborationsWithSuffix={collaborationsWithSuffix}
+          />
+          {' ('}
+          <LiteratureDate wrapperClassName="wrap-in-parentheses" date={date} />
```
Comment (harunurhan) [inline ui/src/literature/components/LiteratureItem.jsx]: `wrapperClassName` is leftover, also you might wanna check if `date` is present before putting parenthesis.

**Pull request merged**